pkg/logs/config/integration_config.go

```diff
@@ -32,8 +32,9 @@ const (
 type LogsConfig struct {
 	Type string
 
-	Port int    // Network
-	Path string // File, Journald
+	Port        int    // Network
+	IdleTimeout string `mapstructure:"idle_timeout" json:"idle_timeout"` // Network
```

Comment:
NIT: comment should be updated

Reply:
How so?  My understanding was that `// Network` indicates which type the config corresponds to..

Reply:
Oh my bad. I thought the comment was used differently. ignore me 😅